src/proxy-ui-api/frontend/src/views/AddClient/TokenPage.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <div>
+    {{$t('wizard.token.info')}}
+    <v-text-field
+      v-model="search"
+      :label="$t('wizard.token.tokenName')"
+      single-line
+      hide-details
+      class="search-input"
+      data-test="token-search-input"
+    >
+      <v-icon slot="append">mdi-magnify</v-icon>
+    </v-text-field>
+
+    <v-radio-group v-model="tokenGroup">
+      <div class="radio-row" v-for="token in filteredTokens" :key="token.id">
+        <v-radio :label="`Token ${token.name}`" :value="token" :disabled="!token.logged_in"></v-radio>
+        <div>
+          <large-button
+            @click="confirmLogin(token)"
+            v-if="!token.logged_in"
+            :disabled="!token.available"
+            data-test="token-login-button"
+          >{{$t('keys.logIn')}}</large-button>
+          <large-button
+            v-if="token.logged_in"
+            outlined
+            disabled
+            data-test="token-logout-button"
+          >{{$t('wizard.token.loggedIn')}}</large-button>
+        </div>
+      </div>
+    </v-radio-group>
+
+    <div class="button-footer">
+      <div class="button-group">
+        <large-button
+          outlined
+          @click="cancel"
+          :disabled="!disableDone"
+          data-test="cancel-button"
+        >{{$t('action.cancel')}}</large-button>
+      </div>
+
+      <div>
+        <large-button
+          @click="previous"
+          outlined
+          class="previous-button"
+          data-test="previous-button"
+        >{{$t('action.previous')}}</large-button>
+
+        <large-button
+          @click="done"
+          :disabled="disableNext"
+          data-test="next-button"
+        >{{$t('action.next')}}</large-button>
+      </div>
+    </div>
+    <TokenLoginDialog :dialog="loginDialog" @cancel="loginDialog = false" @save="tokenLogin" />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import LargeButton from '@/components/ui/LargeButton.vue';
+import TokenLoginDialog from '@/components/token/TokenLoginDialog.vue';
+import { ValidationProvider, ValidationObserver } from 'vee-validate';
+import { Token } from '@/types';
+
+export default Vue.extend({
+  components: {
+    LargeButton,
+    ValidationObserver,
+    ValidationProvider,
+    TokenLoginDialog,
+  },
+  computed: {
+    ...mapGetters(['tokens']),
+
+    filteredTokens: {
+      get(): Token[] {
+        return this.$store.getters.tokensFilteredByName(this.search);
+      },
+    },
+
+    disableSelection() {
+      if (this.tokens.length === 1) {
+        return true;
+      }
+      return false;
+    },
+
+    disableNext() {
+      if (this.tokenGroup) {
+        return false;
+      }
+      return true;
```

Comment:
I think this could also be written as oneliner instead:

return this.tokenGroup ? false : true;

Reply:
Same thing as in the previous one.